doc/footer.html

```diff
@@ -17,5 +17,126 @@
 </a> $doxygenversion
 </small></address>
 <!--END !GENERATE_TREEVIEW-->
+<script type="text/javascript">
+//<![CDATA[
+function myStringCompare(str1, str2) {
```

Comment:
exaplain why you need your own in a comment and cannot use http://www.w3schools.com/jsref/jsref_localecompare.asp